automation_tools/__init__.py

```diff
@@ -1809,8 +1828,18 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
         The url for capsule repo from latest satellite compose.
     TOOLS_URL
         The url for sat-tools repo from latest satellite compose.
+    FROM_VERSION:
+        The satellite current version. To differ representation of
+        Default Organization. e.g '6.1','6.0'
 
     """
+    if os.environ.get('FROM_VERSION') == '6.1':
+        org = '\'Default Organization\''
```

Comment:
Why the need of wrapping in single quotes here? And this not being needed on line 1839?